### DISCUSSION TITLE
I have a drive that is showing in 2 zfs raid groups, how to I fix this?

### DISCUSSION THREAD

**Discussion by avorial** [openzfs/zfs#15849] [Q&A]
SDG shows up in 2 raid groups, It only should be in the first one. How do i remove it from the second, so I can connect the drive that should be there?

![image](https://github.com/openzfs/zfs/assets/65983414/782c26ab-c759-4c98-8d5f-276553d8d461)

**Comment by rincebrain** [comment]
That appears to be some third-party UI, you should probably ask the people providing that UI why it's confused in that fashion.

Offhand, I would guess it's because it gives the "last" name that a missing disk went by, and since you have an "offline" disk, and appear to be using /dev/sdX names, which is what all our guides encourage people not to do because Linux doesn't promise they're the same if anything changes, something else may be using that name, and then it's just showing you the "old" name because it's the best piece of information it's got.
